test/e2e/common/configmap_volume.go

```diff
@@ -540,6 +540,22 @@ var _ = Describe("[sig-storage] ConfigMap", func() {
 		})
 
 	})
+
+	//Slow (~5 mins)
+	It("Should fail non-optional pod creation due to configMap object does not exist [Slow]", func() {
```

Comment:
So... these are good tests but they expose a weird set of failure conditions that are probably not optimal today.  
I think we should almost open issues around some of the failure scenarios here.  

If it's slow, you're basically waiting for the kubelet to bounce vs. failing on admission to sanity check the deps in the manifest.  

I'm ok with the addition, but please open another issue and add a comment block on the details of the failure condition.

Reply:
@timothysc I have created an issue #70937

Reply:
@timothysc added comments for failure cases.